docs/UserGuide.md

```diff
@@ -242,6 +287,7 @@ From left to right, the amenities are **aircon, charger, food, wifi**.
 ### Command Box 
 
 The **Command Box** is where you can enter [**commands**](#how-to-read-commands-in-studytracker) in StudyTracker.
+Similar to any CLI, you can navigate the history of commands you entered with UP and DOWN arrow keys.
```

Comment:
I think this UP and DOWN arrow keys can be put inside a Tip callout.

Then the Tip callout about Managing Themes doesn't seem to fit here anymore. It should go in `## GUI Layout` just before `### Info Display`, I think